### PR TITLE
added endpoint to serve f/e static files

### PR DESCRIPTION
**Pull request by greeendiamond**
please review.

### CODE REVIEW

File: backend/src/app.js [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
+const expressStaticGzip = require('express-static-gzip');
 
 const app = express();
 
 app.use(express.json());
+
+app.use(
+	'/',
+	expressStaticGzip(path.resolve(__dirname, '../../frontend/public'), {
+		enableBrotli: true,
```
Comment (greeendiamond) [inline backend/src/app.js:12]: @soonbinan 
I'm not sure whether we should enable brotli.  Can you compare gzip and brotli and then tell us which one is better in terms of security & performance? 
Thanks.

Comment (soonbinan) [reply backend/src/app.js:12]: according to https://expeditedsecurity.com/blog/nginx-brotli/ and other articles, brotli seems faster and does a better job in compression. right? I don't know about security? tell us about it

Comment (greeendiamond) [reply backend/src/app.js:12]: Brolit-compressed files are only transferred over https. not http. So it's considered more secure way when it comes to compression algorithm. So look it up the difference between https and http. Tell us why https is more secure than http.

Comment (soonbinan) [reply backend/src/app.js:12]: @greeendiamond https://www.cloudflare.com/learning/ssl/why-is-http-not-secure/ explains it well. I knew the basic but took away much more from this. Thanks.

**Pull request merged**